extras/startup-scripts/patroni.service

```diff
@@ -14,7 +14,7 @@ Group=postgres
 # Read in configuration file if it exists, otherwise proceed
 EnvironmentFile=-/etc/patroni_env.conf
 
-WorkingDirectory=~
+# WorkingDirectory=~
```

Comment:
Please also add a comment that by default the User home directory is used and if you want it changed you have to specify an absolute path.